com.cubrid.common.ui/src/com/cubrid/common/ui/cubrid/table/export/handler/ExprotToSqlHandler.java

```diff
@@ -94,6 +94,10 @@ public void exportByQuerying(String tableName) throws IOException, SQLException
 		CUBRIDPreparedStatementProxy pStmt = null;
 		CUBRIDResultSetProxy rs = null;
 
+		if (totalRecord == 0) {
```

Comment:
@seunghun-kim What is your opinion what if we move the `exportConfig.getTotalCount()` under the `if (StringUtil.isEmpty(tableName))` statement with `totalRecord` checking?

```
long totalRecord = exportConfig.getTotalCount(tableName);
if (totalRecord == 0) {
...
```